sdk/cognitivelanguage/azure-ai-language-questionanswering/samples/quickstart.py

```diff
@@ -0,0 +1,227 @@
+# ==========================================
+# Install azure-ai-language-questionanswering package with command
+# ==========================================
+
+# pip install azure-ai-language-questionanswering
+
+# ==========================================
+# Tasks Included
+# ==========================================
+# Create a project
+# Update a project
+# Deploy a project
+# Getting Project data (Sources, QA pairs and synonyms)
+# Get Answer
+# Delete Project
+# ==========================================
+# Further reading
+# General documentation: https://learn.microsoft.com/en-us/azure/cognitive-services/language-service/question-answering/overview
+# Reference documentation: https://learn.microsoft.com/en-us/python/api/overview/azure/ai-language-questionanswering-readme?view=azure-python
+# ==========================================
+
+# Dependencies
+import os
+from azure.core.credentials import AzureKeyCredential
+from azure.ai.language.questionanswering.authoring import QuestionAnsweringAuthoringClient
+from azure.ai.language.questionanswering import QuestionAnsweringClient
+from azure.ai.language.questionanswering import models as qna
+
+
+# Setting Resource Variables
+endpoint = "<ENTER_LANGUAGE_ENDPOINT_HERE>"
+key = "<ENTER_LANGUAGE_KEY_HERE>"
+project_name = "<ENTER_PROJECT_NAME_HERE>"
+
+
+# Defining clients
```

Comment:
I apologize if the request for samples was unclear. We do have samples that cover most of these scenarios already in the repo: https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/cognitivelanguage/azure-ai-language-questionanswering/samples

Would you mind taking a look and adding any authoring samples where you think we have gaps or where existing samples fall short?